package.json

```diff
@@ -30,6 +30,7 @@
   },
   "homepage": "https://github.com/ipfs/interface-ipfs-core#readme",
   "dependencies": {
+    "aegir": "^12.1.0",
```

Comment:
Why change it from devDep to dep?

Reply:
Please see issue #165.  Basically `src\files.js:12:21` depends on it.